internal/probe/probe.go

```
@@ -0,0 +1,100 @@
+			},
+		},
+		CheckRedirect: func(req *http.Request, via []*http.Request) error {
+			return http.ErrUseLastResponse
+		},
+	}
+	defer client.CloseIdleConnections()
+	resp, err := client.Do(req)
+	if err != nil {
+		return Result{}
+	}
+	io.Copy(io.Discard, resp.Body)
```

golangci-lint (2.13.2)

[error] Error return value of `io.Copy` is not checked

(errcheck)